### ISSUE TITLE
Not an issue - Great lib! Can I add fonts myself?

### ISSUE THREAD

**Issue by coltc50** [bitbank2/Multi_OLED#4]
Thank you for your great lib. It's working fine but I want to add own fonts. How can I do this? Is there a converter or so?

**Comment by bitbank2** [comment]
Glad you like it. The fonts are fixed size and intertwined in the code in such a way as to make it hard to change/add them. I would say - you're welcome to fork the library and do as you wish, but I would only merge your new code or fonts if you added some cool new font functionality.

**Comment by coltc50** [comment]
Thank you. That's what I find out also. I only want to add a font with size between the normal and big one to have more options. I'm a newbie at Arduino. I searched to the web to found on which way your fonts are coded but didn't find a hint. Can you give me a hint how to add one?
I'm on a project to make a bike computer with two sd1306 displays. It works great but the normal and small font is hard to read and the big one is to big.

**Comment by bitbank2** [comment]
I wrote the font drawing code to fit the physical layout of the memory. The memory is arranged vertically where the LSB is the top and each row of 128 bytes defines a 128x8 pixel block. Therefore, my fonts are all a multiple of 8 pixels tall. It sounds like you want to create fonts which are in between 2 sizes. This will cause the code to become more complex as you have some pixels shifted into some bytes and some into other. The code I wrote is actually quite simple. If you're having trouble decyphering it, writing code to draw 'medium sized' fonts is probably too challenging.
